File: 장승민/CSharp/210215/carManageProgram/carManageProgram/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace carManageProgram
{
    class Customer
    {
        private String name;
        private String tel;
        private String address;
        Car car;
        public Customer(string name, string tel, string address, Car car)
        {
            this.name = name;
            this.tel = tel;
            this.address = address;
            this.car = car;
     
        }
        public string Name { get => name; set => name = value; }
        public string Tel { get => tel; set => tel = value; }
        public string Address { get => address; set => address = value; }


        public void printCustomerInfo()
        {
            Console.WriteLine("이름: " + name);
            Console.WriteLine("전화번호: " + tel);
            Console.WriteLine("주소: " + address);
            car.printCarInfo();
        }
    }
}

File: 예진희/3 JavaClass/210119_class_singleTone/class0119_samDab2_2/src/com/sec_samDab2_2/carApp/customer/Customer.java
package com.sec_samDab2_2.carApp.customer;

import com.sec_samDab2_2.carApp.car.Car;

public class Customer {
	private String name;
	private String tel;
	private String address;
	private Car car; 
	// 만든 클래스 불러 옴, 많이쓰는 형식, 다른패키지에 있음 import
	// 고객 객체생성 생성시 차 객체도 같이 생성

	// 매개변수 사용하는 생성자
	public Customer(String name, String tel, String address, Car car) {
		super();
		this.name = name;
		this.tel = tel;
		this.address = address;
		this.car = car;
	}

	// getter / setter
	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getTel() {
		return tel;
	}

	public void setTel(String tel) {
		this.tel = tel;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public Car getCar() {
		return car;
	}

	public void setCar(Car car) {
		this.car = car;
	}
	
	public void printCustomerInfo() {
		System.out.println("고객명\t: "+name);
		System.out.println("전화\t: "+tel);
		System.out.println("주소\t: "+address);
		car.printCarInfo();
		System.out.println("--------------------------\n");
	}
}
